mambaSharedFramework/HLSValueTypes.swift

```diff
@@ -266,4 +454,68 @@ public func ==(lhs: HLSCodecArray, rhs: HLSCodecArray) -> Bool {
     return lhs.codecs == rhs.codecs
 }
 
+/// Represents information to assist in view presentation.
+///
+/// Indicates when video content in the Variant Stream requires specialized rendering to be properly displayed.
+public struct HLSVideoLayout: Equatable, FailableStringLiteralConvertible {
+    /// Each specifier controls one aspect of the entry. That is, the specifiers are disjoint and the values for a
+    /// specifier are mutually exclusive.
+    public let layouts: [VideoLayout]
+    /// The client SHOULD assume that the order of entries reflects the most common presentation in the content.
+    ///
+    /// For example, if the content is predominantly stereoscopic, with some brief sections that are monoscopic then the
+    /// Multivariant Playlist SHOULD specify `REQ-VIDEO-LAYOUT="CH-STEREO,CH-MONO"`. On the other hand, if the content
+    /// is predominantly monoscopic then the Multivariant Playlist SHOULD specify `REQ-VIDEO-LAYOUT="CH-MONO,CH-STEREO"`.
+    public let predominantLayout: VideoLayout
+
+    public enum VideoLayout: String {
```

Comment:
its easy (for me) to confuse `HLSVideoLayout` and `VideoLayout`

I wonder if we could consider `VideoLayout` -> `VideoLayoutType` to indicate the enumeration?

Or `HLSVideoLayout` -> `VideoLayoutInformation`? Or something that indicates an overall organization/structure of the layouts available

Reply:
Yeah... Not the best... I'll rename it (maybe `VideoLayoutIdentifier` may be better).

Reply:
Updated to `VideoLayoutIdentifier` here: 440593faa16ffa74b52799acc8a957c5f6eaaf9b